### PR TITLE
[GS-1]Create skeleton of project for desktop

### PR DESCRIPTION
**Pull request by MaksymTimofeev**

### CODE REVIEW

**Comment by MaximDrobchak** [comment]
dfg346463456
Can't resolve this margin for example

**Comment by MaksymTimofeev** [comment]
отступы
I can't indent the left margin

**Comment by MaksymTimofeev** [comment]
При маштабировании сдвигаются блоки
Blocks move when scaling

File: css/style.css [hunk 1]
```diff
@@ -42,13 +42,12 @@
...
-  margin-bottom: 50px;
 }
 
 .marginLeft {
   margin-left: 20px;
 }
 
 .marginBottom {
-  margin-bottom: -50px;
+  margin-bottom: 50px;
```
Comment (MaximDrobchak) [inline css/style.css:52]: 👍

File: img/Rectangle.jpg
Comment (MaximDrobchak) [inline img/Rectangle.jpg:1]: Remove this image it's dublicate and image should not be in the skeleton

File: img/img1.jpg
Comment (MaximDrobchak) [inline img/img1.jpg:1]: Remove this image, The picture should not be in the skeleton

File: img/63b3e5defc6c0510d0dc78a26ec1df10.png
Comment (MaximDrobchak) [inline img/63b3e5defc6c0510d0dc78a26ec1df10.png:1]: Remove this image, The picture should not be in the skeleton

**Review by MaximDrobchak — CHANGES_REQUESTED** [review]

**Comment by MaximDrobchak** [comment]
@MaksymTimofeev  Remove this screenshot, it is not needed
image

**Comment by MaximDrobchak** [comment]
@MaksymTimofeev  Nice job

**Review by MaximDrobchak — APPROVED** [review]

**Pull request merged**